frontend/src/metabase/search/components/SearchFilterModal/filters/TypeFilter.styled.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from "@emotion/styled";
+
+export const TypeCheckboxGroupWrapper = styled.div`
+  display: grid;
```

Comment:
I couldn't use the `Grid` component from Mantine here since it uses `Grid.Col` components to manage columns etc. I would have used Mantine's `SimpleGrid` component here but we haven't merged that one in just yet.